manual.asc

```diff
@@ -845,6 +845,22 @@ Double the scrolling distance when using the eReader hardware buttons
 
 
 == Advanced Features
+[[mathjax]]
+=== MathJax Support
+Mathjax is a modern typesetting library for math and chemistry.  AnkiDroid support Mathjax cards out of the box as of AnkiDroid 2.9.  AnkiDroid follows the Mathjax configuration that Anki Desktop and AnkiMobile use.  Mathjax is configured to expect TeX formatting, with `\(` and `\)` delimiting inline equations and `\[` and `\]` for display equations.
+
+An example (but nonsensical) card could have the front of:
+....
+\(e^{\pi i} = -1\)
+....
+and a back of:
+....
+\(\ce{c8H10N4O2}\)
+....
+For more details, see https://apps.ankiweb.net/docs/manual.html#mathjax-support[Mathjax Support] in the Anki Manual.
+
+Some users added HTML and Javascript to their card styles in order to enable MathJax support in previous versions.  This is no longer necessary.  Unless your configuration was different than Anki's, this extra HTML and Javascript can be removed.  If this doesn't seem to work, it is likely your start and end characters for inlineMath or displayMath are different.  You can migrate your card formatting or keep using your custom Mathjax configuration.
```

Comment:
I'm on the fence on whether "less is more" here and deleting this whole section is good or "less support is better" and keeping it just in case it avoids support requests is better. 

Other than that doubt (which I don't feel strongly about either way) this seems great

Reply:
I do agree with @mikehardy here, I think a short note that previous workarounds (with a link to the reddit page or something) are no longer required on AnkiDroid 2.9+.